packages/ml/CMakeLists.txt

```diff
@@ -16,6 +16,18 @@ TRIBITS_PACKAGE(ML  DISABLE_CIRCULAR_REF_DETECTION_FAILURE)
 # B) Set up package-specific options
 #
 
+IF(${PACKAGE_NAME}_ENABLE_SuperLU)
```

Comment:
@jhux2 Based on our discussion, Trilinos will fail to configure if a user has enabled ML along with SuperLU 5.0 since SuperLU 5.0 is not supported by ML; this can be circumvented by setting the config option `ML_ENABLE_SuperLU:BOOL=OFF`, which I added to the message at failure. 
Please review the changes here to verify that what I did was the proper way to enforce this, and if the message is acceptable.

Reply:
@ndellingwood This looks fine to me.